builds/e2e/templates/nested-parent-vm-setup.yaml

```diff
@@ -1,29 +1,11 @@
 parameters:
   upstream.protocol: 'amqp'
-  testRunnerCount: '1'
+  test.l4DeploymentFileName: ""
+  l3IpAddress: ''
+  test.trackingId: '' 
```

Comment:
Added these parameters to `nested-parent-vm-setup.yaml`. Everything except `test.l4DeploymentFileName` is optional. I tried to default the l4 deployment file to the basic one, but ran into issues because we would need to depend on the upstream protocol which is also a parameter. Devops didn't like that.

I figure it is probably better/ok to be explicit about which deployment file we are using.